### ISSUE TITLE
Feature Request: Multiple printers in one container

### ISSUE THREAD

**Issue by zacgoodwin** [Rdiger-36/bambulab-ams-spoolman-filamentstatus#5]
Right now I have multiple containers running; 1 for my X1C and one for my A1 mini. It’d be nice to be able to have all of my printers run in one docker.

**Comment by Rdiger-36** [comment]
Yeah thats a good idea to use multiple printers with AMS in one container, i think about it, how i could implement it.

**Comment by Rdiger-36** [comment]
I have added a few things. Do you want to test it?

**Comment by Rdiger-36** [comment]
It is now implemented and released in the newest latest version
